src/mbgl/util/tile_cover.cpp

```diff
@@ -124,19 +124,10 @@ std::vector<TileID> tileCover(const TileCoordinate& tl,
     scanTriangle(tl, tr, br, 0, tiles, scanLine);
     scanTriangle(br, bl, tl, 0, tiles, scanLine);
 
-    t.sort();
-    t.unique();
-
-    t.sort([&](const TileID& a, const TileID& b) {
```

Comment:
:-1: My understanding here is we no longer have the "spiral" effect when loading tiles. I tested it here and the tiles are loading from left to right.